packages/react-menu/package.json

```diff
@@ -11,6 +11,8 @@
   },
   "license": "MIT",
   "scripts": {
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node config/normalize-import --output dist/react-menu/src && yarn docs",
     "build": "just-scripts build",
```

Comment:
I'm curious, so on CI, the relative type imports isn't an issue ?

Reply:
that's correct. on CI `lage` takes the job of `tsc` dep resolution when running builds ordered by dep tree relationships.

Reply:
more info documented in JSDoc + TS issue 
![image](https://user-images.githubusercontent.com/1223799/111626241-0e7d7000-87ee-11eb-9e53-af10f6dad1f0.png)